bin/bootstrap/cloud_init.sh

```diff
@@ -1,37 +1,43 @@
 #!/bin/bash
-echo -e "10.0.1.1 puppet\n10.0.1.2 gitlab\n" >> /etc/hosts
+
 puppet_version=$1
 puppet_role=$2
 control_repo=$3
 
-yum install -y git
-git clone $control_repo /tmp/psick
+echo "### Installing git"
```

Comment:
A nice touch would be to run which git before to check is we need to install it.